### DISCUSSION TITLE
Installing a git branch results in a tag being suggested for update.

### DISCUSSION THREAD

**Discussion by sitedyno** [mason-org/mason.nvim#322] [Q&A] [answered]
`:MasonInstall phpactor@master` works great, however when Mason suggests updates it wants to use the latest release tag instead of the latest commit from the master branch. `phpactor     ■ new version available: phpactor/phpactor master -> 0.18.1`

Is there a way for Mason to check for updates on the specified git branch?

**Comment by williamboman** [comment, marked as answer]
Hello! Git installations generally track the latest tag. It's not currently possible to make it track HEAD (or a specific branch for that matter). Should be fairly easy to implement though (installing and tracking HEAD - aka the default branch) - I'm thinking `:MasonInstall phpactor@HEAD`. Will look into it
